validator/client/runner.go

```diff
@@ -187,6 +180,13 @@ func run(ctx context.Context, v iface.Validator) {
 			// Start fetching domain data for the next epoch.
 			if slots.IsEpochEnd(slot) {
 				go v.UpdateDomainDataCaches(ctx, slot+1)
+				go func() {
+					// set proposer settings for next epoch.
+					if err := v.UpdateProposerSettings(ctx, km); err != nil {
+						log.Warnf("Failed to update proposer settings: %v", err)
```

Comment:
if for some reason it doesn't update correctly due to some sort of connection issue that epoch i was thinking it could just be updated in the next epoch. i was afraid of erroring out the whole system here. what do you think?

Reply:
Yeah. I see both sides. But if it's not intermittent then it'd always be warning. I'm ok leaving it as it is